.git-blame-ignore-revs

```diff
@@ -0,0 +1,3 @@
+# .git-blame-ignore-revs
```

Comment:
This line seems unnecessary.

Reply:
I kept the format recommended in the GitHub [documentation](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view).

Reply:
👍